stories/dashboard.stories.js

```diff
@@ -98,6 +99,23 @@ storiesOf( 'Dashboard', module )
 			</WithTestRegistry>
 		);
 	} )
+	.add( 'URL Search Widget', () => {
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( CORE_SITE ).receiveSiteInfo( {
+				usingProxy: true,
+				referenceSiteURL: 'http://example.com',
+				adminURL: 'http://example.com/wp-admin',
+				timezone: 'America/Detroit',
+				siteName: 'My Site Name',
+			} );
+		};
```

Comment:
We can just use `provideSiteInfo` helper:

```suggestion
		const setupRegistry = ( registry ) => {
			provideSiteInfo( registry );
		};
```